backend/app/tests/services/llm/providers/STTproviders/test_STT_SarvamProvider.py

```
@@ -0,0 +1,193 @@
+                print(f"Reasoning Tokens: {result.usage.reasoning_tokens}")
+            # Uncomment to see the raw response:
+            # import json
+            # print("\n--- Raw Provider Response ---")
+            # print(result.provider_raw_response)
+
+    finally:
+        # Clean up the temporary file
+        if temp_audio_file_path and os.path.exists(temp_audio_file_path):
+            os.remove(temp_audio_file_path)
+            print(f"Cleaned up temporary file: {temp_audio_file_path}")
+ 
```

Ruff (0.15.2)

[warning] No newline at end of file

Add trailing newline

(W292)